src/topology/algebra/infinite_sum.lean

```diff
@@ -414,6 +414,26 @@ lemma tsum_congr_subtype (f : β → α) {s t : set β} (h : s = t) :
   ∑' (x : s), f x = ∑' (x : t), f x :=
 by rw h
 
+lemma tsum_zero' (hz : is_closed ({0} : set α)) : ∑' b : β, (0 : α) = 0 :=
+begin
+  classical,
+  rw [tsum, dif_pos summable_zero],
+  have : ∀ (x : α), has_sum (λ (b : β), (0 : α)) x → x = 0,
+  { intros x hx,
+    contrapose! hx,
+    simp only [has_sum, tendsto_nhds, finset.sum_const_zero, filter.mem_at_top_sets, ge_iff_le,
+               finset.le_eq_subset, set.mem_preimage, not_forall, not_exists, exists_prop,
+               exists_and_distrib_right],
+    refine ⟨{0}ᶜ, ⟨is_open_compl_iff.mpr hz, _⟩, λ y, ⟨⟨y, subset_refl _⟩, _⟩⟩,
+    { simpa using hx },
+    { simp } },
+  refine this _ _,
+  exact classical.some_spec _
+end
+
+@[simp] lemma tsum_zero [t1_space α] : ∑'b:β, (0:α) = 0 :=
+tsum_zero' is_closed_singleton
```

Comment:
```suggestion
@[simp] lemma tsum_zero [t1_space α] : ∑' b : β, (0 : α) = 0 := tsum_zero' is_closed_singleton
```
It's up to you if you want to put it in one line or two, and I understand that this is how the theorem was stated previously, but the mathlib style guide requires binders are separated from variables by a space.

Reply:
The suggestion is not good: the proof can be after `:=` only if it fits completely on the line. Adding spaces around the binders is up to the author, sometimes one can remove the spaces if tighter formulas read better.

Reply:
I left the syntax unchanged because that's how it was before, and it makes the git diff seem nicer to me. I've incorporated the comment and made the syntax the same in the two lemmas because I don't want to bikeshed about this.

Reply:
The binder thing is in the style guide. Quoting: "Use a space after binders". Also, I believe to have checked the proof fit in the 100 character limit, but that one is completely optional.

Reply:
Anyways, trivial fixes like these can always be put in their own PRs.

Reply:
But I did make the change according to the guide now!